src/game.cpp

```diff
@@ -13116,53 +13161,116 @@ void game::climb_down( const tripoint &examp )
+                            || m.has_flag( ter_furn_flag::TFLAG_GOES_UP, where ) );
     bool has_grapnel = you.has_amount( itype_grapnel, 1 );
     bool web_rappel = you.has_flag( json_flag_WEB_RAPPEL );
     const int climb_cost = you.climbing_cost( where, examp );
     const float fall_mod = you.fall_damage_mod();
     add_msg_debug( debugmode::DF_IEXAMINE, "Climb cost %d", climb_cost );
     add_msg_debug( debugmode::DF_IEXAMINE, "Fall damage modifier %.2f", fall_mod );
-    const char *query_str;
-    if( !web_rappel ) {
-        query_str = n_gettext( "Looks like %d story.  Jump down?",
-                               "Looks like %d stories.  Jump down?",
-                               height );
-    } else {
-        query_str = n_gettext( "Looks like %d story.  Nothing your webs can't handle.  Descend?",
-                               "Looks like %d stories.  Nothing your webs can't handle.  Descend?", height );
-    }
 
-    if( height > 1 && !query_yn( query_str, height ) ) {
-        return;
-    } else if( height == 1 ) {
+    {
         you.set_activity_level( ACTIVE_EXERCISE );
         weary_mult = 1.0f / you.exertion_adjusted_move_multiplier( ACTIVE_EXERCISE );
 
-        if( has_grapnel ) {
-            if( !query_yn( _( "Use your grappling hook to climb down?" ) ) ) {
+        std::string hint_remaining_fall;
+
+        if( height > 1 ) {
+            // After grappling or climbing down from a 2+ high ledge, the player will free-fall.
+            // We don't assess the expected damage but this simple warning is provided.
+            hint_remaining_fall = "\n";
+            hint_remaining_fall += string_format( n_gettext(
+                    "Even if you climb down safely, you will fall <color_yellow>at least %d story</color>.",
+                    "Even if you climb down safely, you will fall <color_red>at least %d stories</color>.",
+                    height - 1 ), height - 1 );
+        }
+
+        if( can_use_ladder ) {
+            if( !query_yn( _( "Use the ladder to climb down?" ) ) ) {
+                return;
+            } else {
+                has_grapnel = false;
+                web_rappel = false;
+            }
+        } else if( has_grapnel ) {
+            std::string query = _( "Use your grappling hook to climb down?" );
+            query += "%s";
+            if( !query_yn( query.c_str(), hint_remaining_fall ) ) {
                 has_grapnel = false;
             } else {
                 web_rappel = false;
             }
         }
 
-        if( !has_grapnel ) {
-            const char *query;
+        if( !can_use_ladder && !has_grapnel ) {
+            std::string query;
+            std::string hint_slip_risk;
+            std::string hint_fall_damage;
+            std::string hint_climb_back;
             if( web_rappel ) {
-                query = _( "Use your webs to descend?" );
+                query = string_format( n_gettext( "Use your webs to descend?",
```

Comment:
I don't think this is an appropriate use of `n_gettext`. Since the singular form does not have any actual variables in it, it won't get translated in any plural way. The implementation of n_gettext has several places that might assert if a plural form can't be found.

You need to break this into separate translation calls, one for N=1 which uses the singular form exclusively, and one n_gettext call for 2+, even though the 'singular' form will never actually get used here.

Reply:
That's how it worked before.  Makes sense.

Reply:
Which line? I tried to find the original. If it's the same then ignore me cause it clearly works well enough.

Reply:
If you're talking about line 13131, the original included "Looks like %d story." in the singular form, which would be fine.

Reply:
Yeah, the line "Looks like %d story." was in the `n_gettext` call but would never be displayed in practice, as you suggested.  It was there before so I'll just restore it here.  The player will still see the "use your webs to climb down?" message.